x/ibc/04-channel/handler.go

```diff
@@ -75,11 +79,21 @@ func HandleMsgChannelOpenAck(ctx sdk.Context, k keeper.Keeper, channelCap *capab
 		return nil, err
 	}
 
+	clientCtx := client.NewContext()
+	channelRes, err := utils.QueryChannel(clientCtx, msg.PortID, msg.ChannelID, false)
+	if err != nil {
+		return nil, err
+	}
+	channel := channelRes.Channel
+
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			types.EventTypeChannelOpenAck,
 			sdk.NewAttribute(types.AttributeKeyPortID, msg.PortID),
 			sdk.NewAttribute(types.AttributeKeyChannelID, msg.ChannelID),
+			sdk.NewAttribute(types.AttributeKeyConnectionID, strings.Join(channel.ConnectionHops, ", ")),
```

Comment:
It's worth documenting somewhere that the value of this attribute is a comma seperated list of channels, since channels may be established over multiple connection hops.

I suppose we will still want to keep the attribute key the same for consistency on the relayer? As opposed to `connection_ids`?

Reply:
`connection_ids` is also fine and probably more correct. The reason for creating the issue is that we want to extract the events in the form `(event, channel_object)`, where `event` is in {`OpenInit`, `OpenTry`,...} and the `channel_object` has the same structure for all channel events. And we want to reduce as much as possible the number of optional fields. Also, it helps with queries.  For example, for `OpenAck` event we are now able to query the connection(s) or the other chain's channel end without waiting for the channel query on the chain emitting the event.